src/main/java/org/hmhb/kml/DefaultKmlService.java

```diff
@@ -0,0 +1,285 @@
+                                )
+                        )
+                )
+        );
+
+        KmlPlacemark tacoMacAtlanta = new KmlPlacemark(
+                "tacoMacAtl",
+                "Taco Mac",
+                "#purpleBalloon",
+                "<p>This is the original.</p>",
+                new KmlPoint(
+                        "-84.3567636,33.7820263"
+                )
+        );
+
+        KmlPlacemark tacoMacKennesaw = new KmlPlacemark(
+                "tacoMacKsu",
+                "Taco Mac",
+                "#purpleBalloon",
+                "<p>This is near KSU.</p>",
+                new KmlPoint(
+                        "-84.5705701,34.0373673"
+                )
+        );
+
+        List<KmlPlacemark> placemarks = Arrays.asList(
+                bartowCounty,
+                cobbCounty,
+                fultonCounty,
+                tacoMacAtlanta,
+                tacoMacKennesaw
+        );
+
+        return new KmlRoot(
+                new KmlDocument(
+                        "Ryan's example KML",
+                        "Ryan's example KML description",
+                        getStyles(),
+                        placemarks
+                )
+        );
+    }
+
+    private List<KmlStyle> getStyles() {
+        KmlStyle purpleShade = new KmlStyle(
+                "purpleShade",
+                new KmlPolyStyle(
+                        "66b73a67",
+                        "normal",
+                        "1",
+                        "1"
+                )
+        );
+
+        KmlStyle purpleBalloon = new KmlStyle(
+                "purpleBalloon",
+                new KmlIconStyle(
+                        new KmlIcon(
+                                "https://hmhb-ryan.herokuapp.com/images/place.png"
```

Comment:
Use the hmhb.herokuapp.com url and put in a ticket to sweep through the code and make these sorts of things configurable.